dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -204,12 +221,18 @@ def profile(self):
 
         :return:
         """
+        top_k_words = self._top_k_words
+        if top_k_words is None:
+            top_k_words = len(self.word_count.keys())
+
         word_count = sorted(self.word_count.items(),
                             key=lambda x: x[1],
-                            reverse=True)
+                            reverse=True)[:top_k_words]
+
         profile = dict(
             vocab=list(self.vocab_count.keys()),
-            vocab_count=dict(self.vocab_count.most_common()),
+            vocab_count=dict(
+                self.vocab_count.most_common(self._top_k_chars)),
```

Comment:
top_k_chars defaults to None, will this cause an issue?

Reply:
No, by default, most_common takes None value, which means it'll return the whole list.